exporter/datadogexporter/config/config.go

```diff
@@ -406,6 +408,9 @@ func (c *Config) Unmarshal(configMap *config.Map) error {
 	if c.Version != "" {
 		c.warnings = append(c.warnings, fmt.Errorf(deprecationTemplate, "version", "v0.52.0", 8783))
 	}
+	if c.Env != "" {
+		c.warnings = append(c.warnings, fmt.Errorf(deprecationTemplate, "env", "v0.52.0", 9016))
```

Comment:
(the warning on the previous message comes from here)

Reply:
That is super cool :D